src/lib/y2storage/boot_requirements_strategies/zipl.rb

```diff
@@ -110,6 +110,18 @@ def unsupported_boot_disk_error
         )
         SetupError.new(message: error_message)
       end
+
+      # Whether the grub partition is readable
+      #
+      # Even if the /boot partition is encrypted using a technology different from
+      # LUKS1, it is possible to read it if theres is an unencrypted zipl partition.
```

Comment:
We should likely explain that the kernel located in /boot/zipl will take care of activating the device containing /boot, so Grub will find it accessible already.